images/runtime/php/runbase.Dockerfile

```diff
@@ -45,7 +45,6 @@ RUN set -eux; \
 		libtidy-dev \
 		libkrb5-dev \
 		libxslt-dev \
-		unixodbc-dev \
```

Comment:
Just to confirm: we're removing `unixodbc-dev` from being installed in the base image, but will later install a specific version of the package as a part of the PHP version's Dockerfile?

Reply:
That is correct. But this will be reverted once the package source fixes the faulty package.